src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -120,8 +124,14 @@
    {:key :policy
     :render #(react/create-element Policy %)}])
 
-(def top-nav-bar-items
-  (filter (fn [r] (contains? #{:workspaces :methods} (:key r))) routes))
+(defn- get-authenticated-nav-bar-items [state]
+  (if (or (nil? (:curator? @state))
+          (not (:curator? @state)))
+    #{:workspaces :methods}
+    #{:library :workspaces :methods}))
```

Comment:
Rather than passing all of state, I would consider just passing the `:curator?` flag.